server/routerlicious/packages/protocol-base/src/protocol.ts

```diff
@@ -114,11 +114,6 @@ export class ProtocolOpHandler {
                 immediateNoOp = true;
                 break;
 
-            case MessageType.Reject:
```

Comment:
i wonder if we need to keep handling for back compat. could someone have an existing document that has rejection ops?

Reply:
This change could cause a proposal that would have been rejected to instead be approved, which would be a different outcome and is technically non-zero risk.

However, the impacted customer base would be the intersection of customers who use rejection (none known), on documents where the rejection was actually used (a further subset), and in cases where the rejection op didn't make it into the most recent summary (a further subset).

If this occurs, the next time a client loads the document they will believe the proposal to be accepted rather than rejected but will otherwise load normally (since this change makes the reject a no-op).  I expect this should be a non-catastrophic scenario for those clients since rejection already had no strong guarantees (e.g. a connection blip could wipe out your rejection opportunity despite trying to send it).

I didn't think this was a particularly worrying threat so I went ahead with removal.

On the other hand, I've left the `MessageType.Reject` handling in `isSystemMessage` to ensure that we continue to correctly filter it out from the runtime, since that could have more-dangerous results.

Reply:
Chatted with Tony offline -- added a crash here to prevent any eventually inconsistent state.